app/lib/pre_assembly/digital_object.rb

```diff
@@ -77,16 +76,11 @@ def content_md_creation_style
         'Manuscript (image-only)' => :book_as_image,
         'Map' => :map
       }
-      # if this object needs to be registered or has no content type tag for a registered object, use the default set in the YAML file
-      if !project_style[:content_tag_override] || content_type_tag.blank?
-        default_content_md_creation_style
-      else # if the object is already registered and there is a content type tag and we allow overrides, use it if we know what it means (else use the default)
-        content_type_tag_mapping[content_type_tag] || default_content_md_creation_style
-      end
+      content_type_tag_mapping[content_type_tag] || default_content_md_creation_style
     end
 
     def default_content_md_creation_style
-      project_style[:content_structure].to_sym
+      project_style.to_sym
```

Comment:
if project_style is being replaced by content_structure, isn't this line incorrect?  Perhaps fixed in a later commit?

Reply:
ticket created #271